tests/validation/tests/v3_api/test_cli.py

```diff
@@ -279,13 +294,92 @@ def test_cli_multiclusterapp_delete(custom_cluster, remove_cli_resource,
 def test_cli_catalog(admin_cli: RancherCli):
     admin_cli.log.info("Testing Creating and Deleting Catalogs")
     admin_cli.login(CATTLE_TEST_URL, ADMIN_TOKEN)
-    catalog = admin_cli.catalogs.add("https://git.rancher.io/system-charts",
-                                     branch="dev")
+    catalog = admin_cli.catalogs.add(SYSTEM_CHART_URL,
+                                     branch=SYSTEM_CHART_BRANCH)
     assert catalog is not None
     deleted = admin_cli.catalogs.delete(catalog["name"])
     assert deleted
 
 
+@if_test_multicluster
+def test_cluster_removal(custom_cluster, admin_cli: RancherCli):
+    admin_cli.log.info("Testing Cluster Removal")
+    deleted = admin_cli.clusters.delete(custom_cluster.name)
+    assert deleted
+
+
+def test_inspection(rancher_cli: RancherCli):
+    # Test inspect on the default project used for cli tests
+    # Validate it has the expected clusterid, id, type, and active state
+    rancher_cli.log.info("Testing Inspect Resource")
+    resource = rancher_cli.inspect(
+        "project", rancher_cli.default_project["id"],
+        format="{{.clusterId}}|{{.id}}|{{.type}}|{{.state}}")
+    assert resource is not None
+    resource_arr = resource.split("|")
+    assert resource_arr[0] == rancher_cli.default_project["clusterId"]
+    assert resource_arr[1] == rancher_cli.default_project["id"]
+    assert resource_arr[2] == "project"
+    assert resource_arr[3] == "active"
+
+
+def test_ps(custom_workload, rancher_cli: RancherCli):
+    rancher_cli.log.info("Testing rancher ps")
+    # Deploy a workload and validate that the ps command shows it in the
+    # correct namespace with the correct name
+    rancher_cli.switch_context(rancher_cli.DEFAULT_CONTEXT)
+    ps = rancher_cli.ps()
+    expected_value = "{}|{}|nginx|2".format(
+        rancher_cli.default_namespace, custom_workload.name)
+    assert expected_value in ps.splitlines()
+
+
+def test_kubectl(custom_workload, rancher_cli: RancherCli):
+    rancher_cli.log.info("Testing kubectl commands from the CLI")
+    rancher_cli.switch_context(rancher_cli.DEFAULT_CONTEXT)
+    jsonpath = "-o jsonpath='{.spec.template.spec.containers[0].image}'"
+    result = rancher_cli.kubectl("get deploy -n {} {} {}".format(
+        rancher_cli.default_namespace, custom_workload.name, jsonpath))
+    assert result == "nginx"
+
+
+# Note this expects nodes not to be Windows due to usage of ifconfig.me
+@pytest.mark.skip(reason="Fails in Jenkins")
+def test_ssh(rancher_cli: RancherCli):
+    rancher_cli.log.info("Testing ssh into nodes.")
+    failures = []
+    rancher_cli.switch_context(rancher_cli.DEFAULT_CONTEXT)
+    nodes = rancher_cli.nodes.get()
+    rancher_cli.log.debug("Nodes is: {}".format(nodes))
+
+    is_jenkins = False
+    if os.environ.get("RANCHER_IS_JENKINS", None):
+        is_jenkins = True
```

Comment:
Can we move this to the top of the file along with all other global variables? It will be easier to manage all of them

Reply:
Yeah.. this is a work in progress still so this entire test is skipped per suggestion by @sangeethah. This part will probably change a lot